modules/test/playwright/fixtures/usersAndOrganizationsPagesTest.ts

```diff
@@ -5,23 +5,34 @@
 
 import {test} from '@playwright/test';
 
+import {SiteSettingsPage} from '../pages/configuration-admin-web/SiteSettingsPage';
+import {ApplicationsMenuPage} from '../pages/product-navigation-applications-menu/ApplicationsMenuPage';
+import {SiteConfigurationDetailsPage} from '../pages/site-admin-web/SiteConfigurationDetailsPage';
 import {ExportUserDataPage} from '../pages/user-associated-data-web/ExportUserDataPage';
 import {PersonalDataErasurePage} from '../pages/user-associated-data-web/PersonalDataErasurePage';
 import {EditOrganizationPage} from '../pages/users-admin-web/EditOrganizationPage';
 import {EditUserPage} from '../pages/users-admin-web/EditUserPage';
 import {ServiceAccountsPage} from '../pages/users-admin-web/ServiceAccountsPage';
+import {TeamsPage} from '../pages/users-admin-web/TeamsPage';
 import {UserPersonalSitePage} from '../pages/users-admin-web/UserPersonalSitePage';
 import {UsersAndOrganizationsPage} from '../pages/users-admin-web/UsersAndOrganizationsPage';
 
 const usersAndOrganizationsPagesTest = test.extend<{
+	applicationsMenuPage: ApplicationsMenuPage;
 	editOrganizationPage: EditOrganizationPage;
 	editUserPage: EditUserPage;
 	exportUserDataPage: ExportUserDataPage;
 	personalDataErasurePage: PersonalDataErasurePage;
 	serviceAccountsPage: ServiceAccountsPage;
+	siteConfigurationDetailsPage: SiteConfigurationDetailsPage;
+	siteSettingsPage: SiteSettingsPage;
+	teamsPage: TeamsPage;
 	userPersonalSitePage: UserPersonalSitePage;
 	usersAndOrganizationsPage: UsersAndOrganizationsPage;
 }>({
+	applicationsMenuPage: async ({page}, use) => {
+		await use(new ApplicationsMenuPage(page));
+	},
```

Comment:
do we need this inside this fixture? why we cannot use its original fixture to retrieve it in the test?